operations/research-and-development/engineering/team/README.md

```diff
@@ -38,13 +34,12 @@ Work for SET comes from various sources:
 3. The [~Sustained Engineering](https://community-daily.mattermost.com/core/channels/sustained-engineering) channel. Support may use this channel to ask questions or raise urgent issues.
 4. The [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel. This is the place where support and community can ask any type of R&D related questions.
 
-## Triage
-
-SET triages [SET tickets](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=33) asynchronously and also helps the Release Manager to
+When a new issue is raised, it is the SET primary's duty to:
 
-* Help route new unassigned tickets to the appropriate engineering team.
-* Assign bugs to SET when there is no clear engineering team owner.
-* Make sure reported bugs are in fact bugs and recommend turning non-bugs into stories or feature requests.
+1. Understand the customer impact and priority of the issue (if required with the help of the SET lead, or relevant product managers). If the issue is really a feature request or low-urgency bug, a JIRA ticket is filed and assigned to the appropriate team. In that case, the SET responsibilities end here. In case of higher-impact issues, the steps continue:
+2. Investigate to understand the source of the issue.
+3. Prepare a "hand-off package" including (where relevant): logs, Grafana snapshots, and above all a clear description of what the issue is ready to be acted on by the responsible team.
+4. Hand-off the issue to the team who owns the problematic part of the product. Of course, if the issue is in the scope of expertise of the SET engineers, they handle the issue themselves.
```

Comment:
Maybe we should be more specific here too. What does "hand off" actually mean? Are they adding the person to an incident channel? Are they just sending a message to the lead of the team?

What exactly is being handed off? The ownership and communication responsibilities of the incident or just the investigation and fix part?

I think the prescribed we can be the better since receiving any escalation will be a fairly stressful situation for a lot of people. Clear steps for them to follow might make it easier on them and create more consistency.

Reply:
True, I'll clarify this with some of the people who often get such hand-offs to make sure we define what they need.